app/src/app/api/v0/city/[city]/cap/route.ts

```diff
@@ -0,0 +1,27 @@
+import { apiHandler } from "@/util/api";
+import { LANGUAGES } from "@/util/types";
+import { ACTION_TYPES } from "@/util/types";
+import { readFile } from "@/backend/CapService";
+import { NextRequest } from "next/server";
+
+export const GET = apiHandler(async (req: NextRequest, { params, session }) => {
+  if (!session) {
+    throw new Error("Unauthorized");
+  }
```

Comment:
Would be good to check if the user has access to the given city/ inventory/ project/ organization... Not sure if we can do that with just the given parameters though.